src/pages/Product/Product.tsx

```diff
@@ -1,25 +1,49 @@
 import './Product.css';
 // We will need to replace with our actual products
-import { products } from '../../data';
+//import { products } from '../../data';
 import ProductCard from '../../components/ProductCard/ProductCard';
+import { Product, Products } from '../../dataTypes/product';
+import { useEffect, useState } from 'react';
+async function fetchDatafromBack() {
+	const response = await fetch('http://localhost:3000/products');
+
+	if (!response.ok) {
+		throw new Error(`HTTP error! status: ${response.status}`);
+	}
+
+	console.log(response);
+	const data = await response.json();
+	return data;
+}
+
+function ProductList() {
```

Comment:
As you've changed this name, I think the component folder and file name should be updated to `ProductList` too to match convention.